src/plugins/discourse/weights.js

```diff
@@ -63,11 +65,26 @@ export const weightsConfigParser: C.Parser<WeightsConfig> = C.fmap(
   upgrade
 );
 
-export function likeWeight(user: ?User): NodeWeight {
+export function likeWeight(
+  {weights = {}}: DiscourseConfig,
```

Comment:
I think it'll be cleaner to make weights mandatory (see: https://github.com/sourcecred/sourcecred/pull/2643#discussion_r556213209) and just take a definite non-null DiscourseConfig